.vscode/wp-parsely.code-workspace

```diff
@@ -0,0 +1,152 @@
+{
+	"folders": [
+		{
+			"name": "wp-parsely",
+			"path": ".."
+		}
+	],
+	"launch": {
+		// Use IntelliSense to learn about possible attributes.
+		// Hover to view descriptions of existing attributes.
+		// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+		"version": "0.2.0",
+		"configurations": [
+			{
+				"name": "Listen for Xdebug",
+				"type": "php",
+				"request": "launch",
+				"port": 9003
+			},
+		]
+	},
+	"extensions": {
+		"recommendations": [
+			"dbaeumer.vscode-eslint",
+			"usernamehw.errorlens",
+			"xdebug.php-debug",
+			"bmewburn.vscode-intelephense-client",
+			"obliviousharmony.vscode-php-codesniffer",
+			"persoderlind.vscode-phpcbf",
+			"sanderronde.phpstan-vscode"
+		]
+	},
+	"settings": {
+		// VSCode Custom Settings
+		"typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
Why do we need this line? What does it do for us?

Reply:
Instead of using the VSCode provided TS version it uses the one from `node_modules` folder on which project is dependent.